### ISSUE TITLE
Try stdlib=libc++

### ISSUE THREAD

**Issue by jmeyers314** [LSSTDESC/Coord#11]
I get the following when trying to build out-of-the box:

```
[meyers18@Temeria ~/src/Coord (master|…)]$ python setup.py build
Python version =  3.7.2 (default, Dec 29 2018, 00:00:04) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
sources =  ['src/Angle.cpp']
headers =  ['include/Angle_C.h']
Coord version is 1.3
running build
running build_py
creating build
creating build/lib.macosx-10.7-x86_64-3.7
creating build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/_version.py -> build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/util.py -> build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/angle.py -> build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/__init__.py -> build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/celestial.py -> build/lib.macosx-10.7-x86_64-3.7/coord
copying coord/angleunit.py -> build/lib.macosx-10.7-x86_64-3.7/coord
creating build/lib.macosx-10.7-x86_64-3.7/coord/include
copying coord/include/Angle_C.h -> build/lib.macosx-10.7-x86_64-3.7/coord/include
running build_ext
building 'coord._coord' extension
creating build/temp.macosx-10.7-x86_64-3.7
creating build/temp.macosx-10.7-x86_64-3.7/src
gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/meyers18/src/lsst/python/miniconda3-4.5.12/envs/lsst-scipipe-fcd27eb/include -arch x86_64 -I/Users/meyers18/src/lsst/python/miniconda3-4.5.12/envs/lsst-scipipe-fcd27eb/include -arch x86_64 -Iinclude -I/Users/meyers18/src/lsst/python/miniconda3-4.5.12/envs/lsst-scipipe-fcd27eb/include/python3.7m -c src/Angle.cpp -o build/temp.macosx-10.7-x86_64-3.7/src/Angle.o
warning: include path for stdlibc++ headers not found; pass '-stdlib=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
src/Angle.cpp:23:10: fatal error: 'cmath' file not found
#include <cmath>
         ^~~~~~~
1 warning and 1 error generated.
error: command 'gcc' failed with exit status 1
```

If I add the following patch to setup.py though, all is fine:

```
17c17
< ext=Extension("coord._coord", sources, depends=headers, include_dirs=['include'])
---
> ext=Extension("coord._coord", sources, depends=headers, include_dirs=['include'], extra_compile_args=['-stdlib=libc++'], extra_link_args=['-stdlib=libc++'])
56d55
< 
```

This is with apple LLVM 10.0.1:

```
[meyers18@Temeria ~/src/Coord (master|…)]$ gcc --version
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

**Comment by rongpu** [comment]
Thanks @jmeyers314. I had the same error while installing this package for TreeCorr, and your patch fixed it.
I'm running MacOS Catalina with`Apple clang version 11.0.0 (clang-1100.0.33.8)`

**Comment by rmjarvis** [comment]
Fixed via #13
